sceneview/src/main/java/io/github/sceneview/SceneView.kt

```diff
@@ -497,16 +497,16 @@ open class SceneView @JvmOverloads constructor(
             }
         }
 
-        lifecycle.dispatchEvent<SceneLifecycleObserver> {
-            onFrame(frameTime)
-        }
-        onFrame?.invoke(frameTime)
 
         transformManager.commitLocalTransformTransaction()
+        onFrame?.invoke(frameTime)
```

Comment:
I think this line shouldn't be moved since it doesn't affect the `SurfaceMirrorer`.

Reply:
Yes reverted